app/routers/agenda.py

```diff
@@ -47,10 +48,14 @@ def agenda(
     )
     events = defaultdict(list)
     for event_obj in events_objects:
-        event_duration = agenda_events.get_time_delta_string(
-            event_obj.start, event_obj.end
-        )
-        events[event_obj.start.date()].append((event_obj, event_duration))
+        event_duration = agenda_events.get_time_delta_string(event_obj.start,
+                                                             event_obj.end)
+        json_event_data = jsonable_encoder(event_obj)
+        json_event_data['duration'] = event_duration
+        json_event_data['start'] = event_obj.start.time().strftime(
+            "%H:%M")
+        events[event_obj.start.date().strftime("%d/%m/%Y")].append(
```

Comment:
I would separate `event_obj.start.date().strftime("%d/%m/%Y")` to a new line and give it a name, to simplify the reading of this function :)